hashing/src/lib.rs

```diff
@@ -54,11 +64,34 @@ impl Digest {
     pub const SENTINEL_NONE: Digest = Digest([0u8; Digest::LENGTH]);
     /// Sentinel hash to be used by `hash_slice_rfold`. Terminates the fold.
     pub const SENTINEL_RFOLD: Digest = Digest([1u8; Digest::LENGTH]);
-    /// Sentinel hash to be used by `hash_vec_merkle_tree` in the case of an empty list.
+    /// Sentinel hash to be used by `hash_merkle_tree` in the case of an empty list.
     pub const SENTINEL_MERKLE_TREE: Digest = Digest([2u8; Digest::LENGTH]);
 
-    /// Creates a 32-byte BLAKE2b hash digest from a given a piece of data
+    /// Depending on the size of the data it either:
+    /// 1. Creates a 32-byte BLAKE2b hash digest from a given a piece of data, or
+    /// 2. Splits the data into chunks and creates hash digest of the Merkle tree
```

Comment:
Maybe add that this is disabled for now.

Reply:
Hmm, I'd rather revert the documentation to previous state.
We can always update it when we decide it is safe to uncomment the chunking stuff.
What do you say?

Reply:
I'm for reverting the documentation and updating it later.

Reply:
As discussed, I'm restoring the original comment until the new code is in effect.
https://github.com/casper-network/casper-node/pull/2119/commits/43220eddb9ec11af529539bef84b4507b5072218